CHANGELOG.md

```diff
@@ -1,139 +1,146 @@
 # Changelog
 
--   Tested up to: WordPress 6.3.2
+- Tested up to: WordPress 6.3.3
+
+## v1.2.1
+
+### Chore
+
+- Update license to EUPL 1.2
```

Comment:
Ik zou eigenijk liever de changelog zonder de tussenkopjes hebben maar dat is zorg voor later